web/static/js/graph.js

```diff
@@ -242,6 +244,13 @@ Prometheus.Graph.prototype.getEndDate = function() {
   if (!self.endDate || !self.endDate.val()) {
     return null;
   }
+  //
+  // This is returning the time set according to the system clock,
+  // but the datetimepicker is converting this to UTC, which is
+  // causing the time to decrease instead of increase. It should
+  // be increasing by 30min, but UTC is one hour behind berlin time,
+  // so incrementing UTC by 30min e.g. decrements the time the user sees by 30min.
+  //
```

Comment:
Can you look into this? My machine is returning a time that's an hour behind, even though the returned date object claims it is UTC + 1. * sigh * js dates.